pygments/lexers/c_like.py

```diff
@@ -712,8 +711,30 @@ class FlexLexer(RegexLexer):
         ],
         # Handle the rules section
         'rules': [
+            #include('whitespace'),
+            (r'%%', String.Delimiter, ('usercode','#pop')),
+            (r'/\*',Comment.Multiline,'CommentMulti'),
+            (r'\n',Whitespace),
+            (r'[\t\s]+',Whitespace,'cLexer'),
+            (r'"', String, 'string'),
```

Comment:
Copied regex into this section, couldn't come up with a method for using regex by default
tried if (r'\n',Whitespace,'regex') to move to regex at the end of every line but for lines in test where there are line breaks
e.g. %%

"/*"  the state moves to regex after %% but regex pops it back to rules since theres a \n straight after moving